app/src/features/projects/ProjectsRouter.tsx

```diff
@@ -87,6 +87,13 @@ const ProjectsRouter: React.FC<IProjectsRouterProps> = (props) => {
         component={BlockObservationPage}
         componentProps={props}
       />
+      <PrivateRoute
+        exact
+        layout={ProjectsLayout}
+        path="/projects/:id?/surveys/:survey_id?/observations/:observation_id?/block"
```

Comment:
should this say?
path="/projects/:id?/surveys/:survey_id?/observations/:observation_id?/edit" 
or
path="/projects/:id?/surveys/:survey_id?/observations/:observation_id?/details"

Reply:
lets discuss this on call -> no this should be just block cause it is specific for block observations
I have never seen the error above so if you could show me how you reproduced it that would be awesome

Reply:
All good!   I needed to cleanout the database.